internal/controller/ols_console_ui_assets.go

```diff
@@ -112,7 +111,7 @@ func (r *OLSConfigReconciler) generateConsoleUIDeployment(cr *olsv1alpha1.OLSCon
 			Labels:    generateConsoleUILabels(),
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: &replicas,
+			Replicas: cr.Spec.OLSConfig.DeploymentConfig.Replicas,
```

Comment:
@raptorsun Does this make `one replica` count for all our deployments ? We will not be able to control like 1 service Pod , 3 console pod etc ?

Reply:
In that case, we can add a new field to the console config structure.

Reply:
let me post a new version with a separate replica count for console